### DISCUSSION TITLE
Templete purchased with missing pages and elements

### DISCUSSION THREAD

**Discussion by Daniel-DVLoopers** [primefaces/community#547] [PrimeNG Templates] [answered]
Hello,
I don't know if I'm in the right place, but a few weeks ago, I bought the "Verona Layout" template from PrimeNG, and it doesn't have all the pages you show in the preview.
How can I get all the pages?

**Comment by cetincakiroglu** [comment]
Hi,

Which pages are missing and which version are you using?

**Comment by Daniel-DVLoopers** [comment]
Hello,
Thank you for your reply.
I am missing these sections:
- Chat
- The complete Calendar
- Mail Section

If you can help me with that please.

Thank you.

On Nov 26, 2023, at 12:44 AM, Çetin ***@***.***> wrote:



Hi,

Which pages are missing?

**Comment by Daniel-DVLoopers** [comment]
Hello,
Thank you for getting back to me.
I am missing these sections:
- Chat
- The complete Calendar
- Mail Section

Can you help me with that, please?

**Comment by cetincakiroglu** [comment]
Hi,

Which version of Verona you're using. Not all the versions have chat and mail section.

**Comment by Daniel-DVLoopers** [comment]
I download the latest version available.


Sent from my iPhone

On Nov 28, 2023, at 1:38 AM, Çetin ***@***.***> wrote:

﻿

Hi,

Which version of Verona you're using. Not all the versions have chat and mail section.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,
![Screenshot 2023-12-01 at 09 16 47](https://github.com/primefaces/community/assets/69278826/608f0eb3-9dd9-430a-a1fc-4bbfe4229861)

I've downloaded the latest (16.0.0) from the store you can see the folder tree from the screenshot. Mentioned components exist and are located in the apps folder. Also, the live version of the verona is the same as the version downloaded from the store. You can check the live version too, all the mentioned components exist.
